src-tool/Pact/Analyze/Check.hs

```diff
@@ -425,10 +416,9 @@ verifyFunctionInvariants (CheckEnv tables _consts _pDefs moduleData caps gov _de
 
         -- Iterate through each invariant in a single query so we can reuse our
         -- assertion stack.
-        ExceptT $ fmap Right $
-          SBV.query $
-            for2 resultsTable $ \(Located info
-              (AnalysisResult querySucceeds _ prop ksProvs)) -> do
+        res <- ExceptT $ fmap Right $
```

Comment:
```suggestion
        res  :: TableMap [Either CheckFailure (CheckSuccess, [VerificationWarning])] <- ExceptT $ fmap Right $
```

This seems like a useful place for an explicit type, to help readers parse all the `folded . folded` and `fmap . fmap` stuff below.